src/EventListener/FormListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Bolt\EventListener;
```

Comment:
Ah ok. Well what do you think about Kernel and Form subnamespaces under EventListener?